client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-confirmation.tsx

```diff
@@ -0,0 +1,86 @@
+import { Button } from '@automattic/components';
+import { NavigatorHeader } from '@automattic/onboarding';
+import {
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+	__experimentalUseNavigator as useNavigator,
+} from '@wordpress/components';
+import { Icon, image, verse, layout } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import NavigatorTitle from './navigator-title';
+import './screen-confirmation.scss';
+
+interface Props {
+	onConfirm: () => void;
+}
+
+const ScreenConfirmation = ( { onConfirm }: Props ) => {
+	const translate = useTranslate();
+	const { goBack } = useNavigator();
+
+	return (
+		<>
+			<NavigatorHeader
+				title={ <NavigatorTitle title={ translate( 'Great job!' ) } /> }
+				description={ translate( 'Prepare to bring your site to life with your unique content.' ) }
+				onBack={ () => {
+					goBack();
```

Comment:
I found we don't need to call `goBack` here as the `NavigatorBackButton` will trigger the `goBack` function automatically.